vocab/it-IT/Nevermind.voc

```diff
@@ -0,0 +1,7 @@
+lascia stare
+lascia perdere
+stop
```

Comment:
well, this has been sitting for a long time buried in other notifications :\

will 'stop' in this skill interfere with an English fallback for stopping media playback?